modules/cf-core/src/protocol/install-virtual-app.ts

```diff
@@ -1171,11 +1168,10 @@ async function getUpdatedStateChannelAndVirtualAppObjectsForIntermediary(
     tokenAddress
   } = params as InstallVirtualAppProtocolParams;
 
-  const stateChannelWithAllThreeParties = getOrCreateStateChannelWithUsers(
+  const stateChannelWithAllThreeParties = await getOrCreateStateChannelWithUsers(
```

Comment:
note that if this function is called and the network has a provider, it may not enforce the multisig is pulled from the store

Reply:
it *looks* like that has been accounted for by use of the static function though, noice